File: benchmark/results/java-multilang/run-004/a-mcp/output/ordering/services/OrderValidator.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;

import java.util.List;
import java.util.Optional;

// OrderValidator service.

public class OrderValidator {

    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    public Optional<Order> findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/a-mcp/output/ordering/services/order_validator.py
from typing import List, Optional
from ordering.aggregates.order import Order

"""OrderValidator service."""
class OrderValidator:
    def create(self, input: Order) -> Order:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Order]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> List[Order]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

